packages/vscode-extension/src/commandHandlers.ts

```diff
@@ -1,11 +1,14 @@
+import { ConnnectionResult } from '@neo4j-cypher/schema-poller/dist/cjs/src/schemaPoller';
```

Comment:
Nitpick. This import has a weird shape. I think it shouldn't be importing directly from the `dist/cjs` folder and look more like:

```
import { ConnnectionResult } from '@neo4j-cypher/schema-poller/schemaPoller';
```

Reply:
ah, well spotted - I'll get this sorted